pex/interpreter.py

```diff
@@ -252,7 +288,7 @@ def iter_tags():
         env_markers = MarkerEnvironment(**values.pop("env_markers"))
         return cls(
             version=cast("Tuple[int, int, int]", version),
-            pypy_version=pypy_version,
+            pypy_version=cast("Optional[Tuple[int, int, int]]", pypy_version),
```

Comment:
Is this change and the `__format_version__` handling related to the attrs fix? If so, can you explain how it connects? Thanks.

Reply:
There was already logic to bust the cache when a new INTERP-INFO field was added (see the comparison of # entries to 16 on LHS of diff), but in this case, the number of entries did not change (well, it did because I added this new version format field, but ignore that) but the contents of an existing entry, `sys_path`, did. The format version field allows busting the cache when no new entries are added, but contents has changed / a big fixed and it's desired to have the old cache entries auto-evicted.